Automattic/wp-calypso
# Display domain ContactVerificationCard only if the domain requires Nominet verification (or it has been suspended)

## Proposed Changes

This PR add extra conditions to show the domain card `ContactVerificationCard`.

The card is displayed only if the Nominet has request a contact verification for the domain or if it has been already suspended.

## Testing Instructions

- Apply this patch locally the visit the management page for a domain you own it that has one of the 2 Nominet properties (pending verification or suspended) active. You can easily mock those on your sandbox (or in React dev tool)
- Verify that the Contact Verification Card is displayed in domain management page
- Disable the properties and check that the card is no longer displayed

Note: You also need to enable `contact-verification-feature` feature flag to enable the card.

## Pre-merge Checklist
